base_layer/core/src/proof_of_work/difficulty.rs

```diff
@@ -115,17 +115,20 @@ impl Default for Difficulty {
 
 /// This trait is used to add a type to `CheckedAdd`, which greatly simplifies usage in the code.
 /// It is implemented for `Difficulty` and `u64`.
+#[cfg(feature = "base_node")]
 pub trait CheckedAdd<T> {
     fn checked_add(&self, other: T) -> Option<Self>
     where Self: Sized;
 }
 
+#[cfg(feature = "base_node")]
```

Comment:
why are all of these suddenly required?

Reply:
It is for `cargo check --package minotari_wallet_ffi`